### PR TITLE
Autoload the AppKernel using Composer

### LINKED ISSUE #868: Why is AppKernel not loaded by Composer?
Is there a reason why the `AppKernel` file is not loaded by Composer? I.e. registered like this in `composer.json`:

```json
{
    ...
    "autoload": {
        ...
        "files": [
            "app/AppKernel.php"
        ]
    },
    ...
}
```

Since it is always loaded, letting Composer do the job would make each entry point of the application (console, app.php, app_dev.php, …) a little simpler.

### PR DESCRIPTION
**Pull request by mnapoli**
Fixes #868

Using Composer to load the AppKernel means we don't have to manually include it in all the entry points of the application.

### CODE REVIEW

**Comment by Tobion** [comment]
:+1:

File: app/console [hunk 1]
```diff
@@ -15,7 +15,6 @@ set_time_limit(0);
  * @var Composer\Autoload\ClassLoader $loader
  */
 $loader = require __DIR__.'/../app/autoload.php';
```
Comment (xabbuh) [inline app/console]: This looks like it can be simplified too.

Comment (mnapoli) [inline app/console]: This is not part of this pull request, additionally I believe the variable is kept so that you can uncomment the code below: https://github.com/symfony/symfony-standard/blob/2.8/web/app.php#L11-L19

**Comment by Tobion** [comment]
ping @symfony/deciders

**Comment by aitboudad** [comment]
@mnapoli I think you missed https://github.com/symfony/symfony-standard/blob/2.8/web/app_dev.php#L27

File: web/app_dev.php [hunk 2]
```diff
@@ -24,8 +24,6 @@
 $loader = require __DIR__.'/../app/autoload.php';
 Debug::enable();
 
-require_once __DIR__.'/../app/AppKernel.php';
```
Comment (hacfi) [inline web/app_dev.php]: @aitboudad it’s there

**Comment by mnapoli** [comment]
@aitboudad see https://github.com/symfony/symfony-standard/pull/869#discussion_r42759424

File: web/app_dev.php (hunk 2 above)
Comment (aitboudad) [inline web/app_dev.php]: :)

**Comment by aitboudad** [comment]
:+1:

**Comment by xabbuh** [comment]
:+1:

**Comment by Tobion** [comment]
Thank you @mnapoli.

**Comment by xabbuh** [comment]
Shouldn't we backport this to older branches too (I don't see a reason why new Symfony 2.3 or 2.7 applications should not benefit from this change).

**Comment by Tobion** [comment]
It's not a bugfix, so IMO it shouldn't be backported.
